runtime/cudaq/builder/kernel_builder.cpp

```diff
@@ -874,10 +911,11 @@ jitCode(ImplicitLocOpBuilder &builder, ExecutionEngine *jit,
   pm.addPass(cudaq::opt::createGenerateDeviceCodeLoader(/*genAsQuake=*/true));
   pm.addPass(cudaq::opt::createGenerateKernelExecution());
   optPM.addPass(cudaq::opt::createLowerToCFGPass());
-  // We want quantum allocations to stay where they are if we are simulating and
-  // have user-provided state vectors. This check could be better / smarter
-  // probably, in tandem with some synth strategy to rewrite initState with
-  // circuit synthesis result
+  // We want quantum allocations to stay where they are if
```

Comment:
`clang-format` gets dumb about multiple line comments, particularly if it gets confused and starts over indenting code.

To fix these, you can convert the comment to a single line and re-run clang-format.